common/src/models/request/organization/organizationSsoRequest.ts

```diff
@@ -0,0 +1,4 @@
+export class OrganizationSsoRequest {
+    enabled: boolean = false;
+    data: any;
```

Comment:
Is there some specific reason this needs to be `any` type? The type should be well-defined on the c# side, I'd prefer to use typescripts static-typing power whenever possible.